app/Services/Minio.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Services;
+
+class Minio extends BaseService
+{
+    protected $imageName = 'minio';
+    protected $defaultPort = 9000;
+    protected $prompts = [
+        [
+            'shortname' => 'volume',
+            'prompt' => 'What is the Docker volume name?',
+            'default' => 'minio_data',
+        ],
+    ];
+
+    protected $dockerRunTemplate = '-p "$port":9000 \
+        -e MINIO_ACCESS_KEY=minio
```

Comment:
I've never used Minio; is it likely people will want to change their access key and secret? If so, can you add those as prompts as well?

Reply:
Yeah, I guess it makes sense to put as prompts with default values. Minio is pretty cool for having a local S3 for development.

Reply:
Updated & pushed.